packages/syft/src/syft/grid/duet/__init__.py

```diff
@@ -295,4 +305,21 @@ def join_duet(
     info("♫♫♫ > " + bcolors.OKGREEN + "CONNECTED!" + bcolors.ENDC, print=True)
     # begin_duet_client_logger(duet.node)
 
+    #TODO (flight): handle outside localhost (hard)
+    #TODO (flight): get available ports
+
+    flight_args = {
```

Comment:
We will need support for TLS, in Duet, WebRTC validates data through self-signed certificates I believe. It would be interesting to validate this. (Don't address in this PR, but we need to keep an eye opened on this)